azure/Guidelines.md

```diff
@@ -225,24 +263,31 @@ Your service should include the `x-ms-request-id` value in error logs so that us
 ### REpresentational State Transfer (REST)
 REST is an architectural style with broad reach that emphasizes scalability, generality, independent deployment, reduced latency via caching, and security. When applying REST to your API, you define your service’s resources as a collections of items.
 These are typically the nouns you use in the vocabulary of your service. Your service's [URLs](#uniform-resource-locators-urls) determine the hierarchical path developers use to perform CRUD (create, read, update, and delete) operations on resources. Note, it's important to model resource state, not behavior.
-There are patterns, later in these guidelines, that describe how to invoke behavior on your service. See [this article in the Azure Architecture Center](https://docs.microsoft.com/en-us/azure/architecture/best-practices/api-design) for a more detailed discussion of REST API design patterns.
+There are patterns, later in these guidelines, that describe how to invoke behavior on your service. See [this article in the Azure Architecture Center](https://docs.microsoft.com/azure/architecture/best-practices/api-design) for a more detailed discussion of REST API design patterns.
 
 When designing your service, it is important to optimize for the developer using your API.
 
+<a name="rest-clear-naming"></a>
 :white_check_mark: **DO** focus heavily on clear & consistent naming
 
+<a name="rest-paths-make-sense"></a>
 :white_check_mark: **DO** ensure your resource paths make sense
 
+<a name="rest-simpify-operations"></a>
 :white_check_mark: **DO** simplify operations with few required query parameters & JSON fields
 
+<a name="rest-specify-string-value-constraints"></a>
 :white_check_mark: **DO** establish clear contracts for string values
 
+<a name="rest-use-standard-status-codes"></a>
 :white_check_mark: **DO** use proper response codes/bodies so customer can diagnose their own problems and fix them without contacting Azure support or the service team
 
-#### JSON Resource Schema & Field Mutability
+#### Resource Schema & Field Mutability
```

Comment:
Here's another little change I made, to distinguish this section from the one focused purely on JSON format.

Reply:
Might want to define an explicit bookmark in case people have referred to it. I imagine both that and the auto-generated one will be available.